services/scripts/build_libs.sh

```diff
@@ -16,6 +16,7 @@ LIBRARIES=(
   "conversations"
   "integrations"
   "alerting"
+  "opensearch"
```

Comment:
Also add it to watch_libs.sh script. It's a map with <lib-name>: @crowd dependencies list of that lib so that the script knows what to rebuild. In you case it should be just: opensearch: types because you only depend on @crowd/types library.